lightkurve/search.py

```diff
@@ -421,6 +424,10 @@ def search_targetpixelfile(target, radius=None, cadence='long',
         'long' or 'short'.
     mission : str, list of str
         'Kepler', 'K2', or 'TESS'. By default, all will be returned.
+    provenance_name : str, list of str
```

Comment:
Can you add what community products are supported? Not all of the HLSPs are supported in this PR.

Reply:
Done!